app/src/components/group_schema_table.rs

```diff
@@ -154,7 +155,35 @@ impl GroupSchemaTable {
         let hardcoded = ctx.props().hardcoded;
         html! {
             <tr key={attribute.name.clone()}>
-                <td>{&attribute.name}</td>
+                {
+                    if hardcoded {
```

Comment:
Nit: I usually try to keep the indentation at 2 spaces for html macros, since they get pretty deep. It's not enforced by cargo fmt, sadly.

Reply:
Check, I'll get it converted.